### PR TITLE
[GLOSSARY] hot wallet

### PR DESCRIPTION
**Pull request by MaxHillebrand**
adds hot wallet to glossary.

I'm not sure if it is right to say that wasabi is a hot wallet "by default" - but I mean it is unless you connect a hww...

### CODE REVIEW

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -51,6 +51,13 @@ A hardware wallet is a special type of Bitcoin wallet which generates and stores
 Read more: [Hardware Wallet FAQs](/FAQ/FAQ-UseWasabi.md#hardware-wallet)
 :::
 
+:::details
+### Hot Wallet
+
+A hot wallet is a software wallet that runs on a computer which is connected to the internet.
```
Comment (SimoneBovi) [inline docs/glossary/Glossary.md]: A hot wallet is a software wallet that runs on a computer which is connected to the Internet.

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
A hot wallet is a software wallet that runs on a computer which is connected to the Internet.
```

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -51,6 +51,13 @@ A hardware wallet is a special type of Bitcoin wallet which generates and stores
 Read more: [Hardware Wallet FAQs](/FAQ/FAQ-UseWasabi.md#hardware-wallet)
 :::
 
+:::details
+### Hot Wallet
+
+A hot wallet is a software wallet that runs on a computer which is connected to the Internet.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
A hot wallet is a software wallet that runs on a machine which is connected to the Internet.
```

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]: is there any machine that is not a computer that can be connected to the internet?

Comment (yahiheb) [reply docs/glossary/Glossary.md]: a mobile?

**Pull request merged**